### PR TITLE
updated helm chart values to include the host and port and the flagd …

### PR DESCRIPTION
**Pull request by bornav**
added helm chart update to include the changes in this PR https://github.com/open-telemetry/opentelemetry-demo/pull/1486

### CODE REVIEW

**Review by julianocosta89 — APPROVED** [review]

**Review by puckpuck — APPROVED** [review]

**Comment by bornav** [comment]
@puckpuck  Hi, have a question regarding the Lint Charts test that failed
Rrunning locally both the generate-examples and the check-examples with the flag CHARTS=opentelemetry-demo, both make command pass without any errors.
Is the error happening right now expected or should something be reworked?

**Comment by TylerHelmuth** [comment]
@bornav make sure you have all the chart dependency repos added locally and then try `make generate-examples CHARTS=opentelemetry-demo` again.

File: charts/opentelemetry-demo/examples/bring-your-own-observability/rendered/opensearch/poddisruptionbudget.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 # Source: opentelemetry-demo/charts/opensearch/templates/poddisruptionbudget.yaml
-apiVersion: policy/v1
+apiVersion: policy/v1beta1
```
Comment (puckpuck) [inline charts/opentelemetry-demo/examples/bring-your-own-observability/rendered/opensearch/poddisruptionbudget.yaml:3]: this change is peculiar. It seems like your local version of kubectl is for [K8s version 1.21 or earlier](https://github.com/opensearch-project/helm-charts/blob/main/charts/opensearch/templates/poddisruptionbudget.yaml#L2-L6)

Upgrading your local kubectl / helm should fix this, and allow for the linter to pass.

Comment (bornav) [reply charts/opentelemetry-demo/examples/bring-your-own-observability/rendered/opensearch/poddisruptionbudget.yaml]: good catch, have addressed the problem, but what's weird is my kubectl version is GitVersion:"v1.27.11", have also tried with 1.29 but still the same
And was only able to build it without v1beta on a different host

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**